File: matlab/utils/norm_percentile_nocrop.m
function [a,normalizacia]=norm_percentile_nocrop(a,perc)

normalizacia=[double(prctile(a(:),perc*100)) double(prctile(a(:),100-perc*100))];
a = (a - normalizacia(1))/(normalizacia(2) - normalizacia(1)) - 0.5;

% a=mat2gray(a,normalizacia);

File: python/utils/norm_percentile_nocrop.py
import numpy as np

def norm_percentile_nocrop(data,perc):
    
    norm = [np.percentile(data,perc*100),np.percentile(data,100-perc*100)]
    data = (data - norm[0])/(norm[1] - norm[0]) - 0.5;
    
    
    return data
    